sdk/glossary.md

```diff
@@ -0,0 +1,36 @@
+# Ably SDK Team: Glossary
+
+Here we define terminology used by the SDK Team at Ably.
```

Comment:
Should we also specify that this is aimed at being accessible for non-technical people?

Reply:
Good idea. I will expand.

Reply:
https://github.com/ably/engineering/pull/35/commits/299e373f80f141b05f4fb380a3aee4204c51b359